apps/entity-service/src/entities/dto/project-pitch.dto.ts

```diff
@@ -0,0 +1,470 @@
+import { JsonApiDto } from "@terramatch-microservices/common/decorators";
+import { ApiProperty } from "@nestjs/swagger";
+import { IsArray, IsDate, IsNumber, IsOptional, IsString, IsUUID, Max, Min } from "class-validator";
+import { JsonApiAttributes, pickApiProperties } from "@terramatch-microservices/common/dto/json-api-attributes";
+import { ProjectPitch } from "@terramatch-microservices/database/entities";
+
+@JsonApiDto({ type: "projectPitches" })
+export class ProjectPitchDto extends JsonApiAttributes<ProjectPitchDto> {
+  constructor(data: ProjectPitch) {
+    pickApiProperties(data, ProjectPitchDto);
+    super({
+      ...pickApiProperties(data, ProjectPitchDto)
+    });
+  }
+
+  @ApiProperty()
+  id: number;
```

Comment:
Is this ID needed on the client? We try to avoid sending numerical IDs when possible, and they usually aren't.